types/v2/types.benc.go

```diff
@@ -0,0 +1,258 @@
+// Code generated by bencgen golang. DO NOT EDIT.
+// source: types.benc
+
+package v2
+
+import (
+    "github.com/deneonet/benc/std"
```

Comment:
Should we put somewhere a bit of context for future selves about why we choose benc and what it was tested against?